pkg/services/org/orgimpl/store.go

```diff
@@ -239,8 +241,16 @@ func (ss *sqlStore) Delete(ctx context.Context, cmd *org.DeleteOrgCommand) error
 			"DELETE FROM alert WHERE org_id = ?",
 			"DELETE FROM annotation WHERE org_id = ?",
 			"DELETE FROM kv_store WHERE org_id = ?",
+			"DELETE FROM team WHERE org_id = ?",
```

Comment:
Quick question have you checked with the backend squad if that's the way to go?
Asking before this seems to diverge from the pattern they set up not so long ago of placing the database delete functions in the target package (ex: DeleteUserPermissions in accesscontrol)

Reply:
I agree that it diverges from the pattern. But if I remember correctly, I tried to do i the "right" way a while ago, and it wasn't trivial to split the deletes into other stores due to circular dependencies :/